python/lsst/ip/isr/isrMock.py

```diff
@@ -571,14 +585,16 @@ def getExposure(self, isTrimmed=None):
         else:
             _isTrimmed = isTrimmed
 
-        camera = self.getCamera()
+        camera = self.getCamera(isForAssembly=self.config.isLsstLike)
```

Comment:
Could this just be True all the time since this is testing assembly? The option has no effect when lsstlike is false, and I think that will be clearer. Also add a comment “we are testing assembly here so set this to True”